### PR TITLE
README.rst: Rectify links of SDKs (Java, Swift)

### PR DESCRIPTION
**Pull request by nanuda**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,4 +32,3 @@ example/db.sqlite
 /wheelhouse
 /test_cli/
 Gemfile.lock
-.idea/
```
Comment (mattrobenolt) [inline .gitignore]: Can you add this back please? It's entirely unrelated to this change.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]
Fix the .gitignore, and will merge away.

File: .gitignore (hunk 1 above)
Comment (nanuda) [inline .gitignore]: I'm new to github. Do I have to close this PR and reopen new PR with reset?

**Review by mattrobenolt — APPROVED** [review]

**Comment by mattrobenolt** [comment]
Thanks! :)

**Pull request merged**